src/runes.c

```diff
@@ -25,7 +26,7 @@ void DoDropRune(int rune, qbool s)
 	item->s.v.velocity[2] = 400;
 	item->s.v.flags = FL_ITEM;
 	item->s.v.solid = SOLID_TRIGGER;
-	item->s.v.movetype = MOVETYPE_TOSS;
+	item->s.v.movetype = (int) cvar("k_ctf_rune_bounce") & 1 ? MOVETYPE_BOUNCE : MOVETYPE_TOSS;
```

Comment:
why not a simple `(cvar("k_ctf_rune_bounce"0) == 1) ? MOVETYPE_BOUNCE : MOVETYPE_TOSS`?

Reply:
because 3 enables them both but I will create a specific PR for these later, accidental push.